weeklymission2/src/Components/FolderHeader.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import logo from "../images/logo.svg";
+import linkIcon from "../images/link.svg";
+
+export function FolderHeader({ profile }) {
+  return (
+    <>
+      <div className="headerArea">
+        <nav>
+          <div className="navBar">
+            <a href="index.html">
+              <img src={logo} alt="logo" className="logo" />
+            </a>
+            {profile.email ? (
+              <div className="profile">
+                <img
+                  src={profile.image || ""}
+                  alt="userProfileImg"
+                  className="profileImg"
+                ></img>
+                <span className="userEmail">{profile.email}</span>
+              </div>
+            ) : (
+              <a href="../Pages/signIn.html" className="btn btnS">
+                로그인
+              </a>
+            )}
+          </div>
+          <form>
+            <label className="linkInputArea">
+              <input
+                placeholder="링크를 추가해 보세요"
+                className="addLinks"
+              ></input>
+              <img src={linkIcon} alt="link icon" className="linkIcon"></img>
+              <button className="addBtn">추가하기</button>
+            </label>
```

Comment:
lable은 `input` 태그만 감싸주세요 .
```suggestion
            <label className="linkInputArea">
              <input
                placeholder="링크를 추가해 보세요"
                className="addLinks"
              ></input>
            </label>
                          <img src={linkIcon} alt="link icon" className="linkIcon"></img>
              <button className="addBtn">추가하기</button>

```

Reply:
label로 세 태그를 감싼 이유는 Input 안에 링크아이콘과 추가하기버튼을 position 으로 고정시키기 위함이었습니다. label 대신 div로 묶었습니다. 혹시 틀리면 다음 리뷰때 말씀해주세요!!

Reply:
넵 리뷰에서 말씀드리도록 할께요 !